robot1_state_controller:
  type: za_control/ZaStateController
  arm_id: robot1
  joint_names:
    - rob1_joint_1
    - rob1_joint_2
    - rob1_joint_3
    - rob1_joint_4
    - rob1_joint_5
    - rob1_joint_6
  publish_rate: 30 # [Hz]

robot2_state_controller:
  type: za_control/ZaStateController
  arm_id: robot2
  joint_names:
    - rob2_joint_1
    - rob2_joint_2
    - rob2_joint_3
    - rob2_joint_4
    - rob2_joint_5
    - rob2_joint_6
  publish_rate: 30 # [Hz]

robot3_state_controller:
  type: za_control/ZaStateController
  arm_id: robot3
  joint_names:
    - rob3_joint_1
    - rob3_joint_2
    - rob3_joint_3
    - rob3_joint_4
    - rob3_joint_5
    - rob3_joint_6
  publish_rate: 30 # [Hz]